neqo-transport/src/cid.rs

```diff
@@ -314,6 +314,10 @@ impl ConnectionIdEntry<[u8; 16]> {
         stats.new_connection_id += 1;
         true
     }
+
+    pub fn is_empty(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_zero_len(&self) -> bool {
```

RFC 9000 refers to this as "zero-length Connection ID", right? How about adopting that wording here?

Reply:
We seem to calling these "empty", e.g., `CONNECTION_ID_SEQNO_EMPTY` so I stuck with that. (You're right that that's different from the RFC.)